src/Views/LanguageView.vala

```diff
@@ -229,6 +236,37 @@ public class Installer.LanguageView : AbstractInstallerView {
         next_step ();
     }
 
+    private void on_next_button_secondary_clicked () {
+        var dialog = new Granite.MessageDialog.with_image_from_icon_name (
+            _("Unexpected click"),
+            _("You clicked the right mouse button where a left-click was expected. Do you want to switch left and right mouse buttons?"),
+            "input-mouse",
+            Gtk.ButtonsType.CANCEL
+        );
+        dialog.transient_for = (Gtk.Window) get_toplevel ();
+
+        var switch_button = dialog.add_button (_("Switch mouse buttons"), Gtk.ResponseType.ACCEPT);
+        switch_button.get_style_context ().add_class (Gtk.STYLE_CLASS_SUGGESTED_ACTION);
+        switch_button.grab_focus ();
```

Comment:
Instead of "Switch" or "Cancel" we probably should be more explicit that the choice is "Left-Click as Primary" or "Right-Click as Primary" or something like that. Any thoughts on how to phrase that better @elementary/ux?

I also wonder if we should have a suggested action here. On the one hand, I want to make sure we present both choices as equally valid, but on the other hand we are showing this because of a right click, so it might make sense that right is suggested because it's in line with the implication

Reply:
Continuing from my comment above, I'm fine with the suggestion since we're only throwing this in response to the right click in the first place.

Is this too obnoxious?

```suggestion
            Gtk.ButtonsType.CANCEL
        );
        dialog.transient_for = (Gtk.Window) get_toplevel ();

        var switch_button = dialog.add_button (_("Use Right Mouse Button as Primary"), Gtk.ResponseType.ACCEPT);
        switch_button.get_style_context ().add_class (Gtk.STYLE_CLASS_SUGGESTED_ACTION);
        switch_button.grab_focus ();
```

Reply:
That seems quite long :sweat_smile: I think just to clarify I'd also like to avoid "Cancel" here since that seems a little bit of an odd response to this question

Reply:
What do you think of this?

![Bildschirmfoto von 2021-12-22 16 07 04](https://user-images.githubusercontent.com/10796736/147113453-4f0cc988-bcc3-4311-aadd-233ce138fe7e.png)

Left and right click is possible on both buttons.

Reply:
That looks great!